Data_Collection/CRON_scripts/foph_fetch.py

```diff
@@ -5,15 +5,15 @@
 import logging
```

Comment:
In the `make_forecast.py` script I have used the [loguru library](https://loguru.readthedocs.io/en/stable/resources/migration.html) to handle logging. it removes a bit of the boilerplate code we need when using logging. Do you think we could use it here as well?

Reply:
I will take a look into that! thanks for pointing me to that.